config/prow/config.yaml

```diff
@@ -466,6 +466,12 @@ branch-protection:
             - stage-bots
         website:
           protect: true
+          enforce_admins: false
```

Comment:
This one is concerning because it would allow bypass of CLA.

What kinds of checks are failing that we would need admins to have a green merge button and not go through tide?

Reply:
We had this happen for v1.28: a contributor whose CLA checks had previously passed changed his details within GitHub, which meant that the commits we'd merged into _dev-1.28_ weren't seen (by the CLA bot) as valid to merge from _dev-1.28_ into _main_.

We **always** bypass Tide for the actual docs change on release day, but we still need a green button to click.

Reply:
Should we allow admins to bypass this rule? I can't decide, but I do trust repository admins to use this privilege only with care.